File: openapi-client/php-dt/src/App/DTO/EmailAddressParam.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * A single email address parameter used as the body of &#x60;POST /mail/blocks/delete&#x60;.
 */
class EmailAddressParam
{
    /**
     * The email address to delist from all block lists.
     */
    #[DTA\Data(field: "email")]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $email = null;

}

File: openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/EmailAddressParam.scala

package org.openapitools.client.model


case class EmailAddressParam (
    /* The email address to delist from all block lists. */
    _email: String
)
object EmailAddressParam {
    def toStringBody(var_email: Object) =
        s"""
        | {
        | "email":$var_email
        | }
        """.stripMargin
}
